ansible/roles/master/templates/kube-apiserver-dropin.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
+After=etcd2.service
+
+[Service]
+User=root
+ExecStart=
```

Comment:
According to CoreOS, the 1st empty ExecStart is needed when you use a dropin that uses ExecStart.I confirmed that it's needed. I will add a comment so people are aware of the issue.